Makefile

```diff
@@ -15,6 +15,7 @@ VERSION=5.0.0-dev
 
 DOCKER_IMAGE ?= quay.io/gravitational/teleport
 DOCKER_IMAGE_CI ?= quay.io/gravitational/teleport-ci
+DOCKER_CACHE_OPTS ?= --no-cache
```

Comment:
This helps iterate faster during development by avoiding excessive image rebuilds.